target/linux/ipq40xx/patches-4.14/901-arm-boot-add-dts-files.patch

```diff
@@ -29,6 +29,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
  	qcom-ipq4019-ap.dk01.1-c1.dtb \
 +	qcom-ipq4019-a62.dtb \
 +	qcom-ipq4019-ap.dk04.1-c1.dtb \
++	qcom-ipq4019-eap2200.dtb \
```

Comment:
Drop this as there is no 4.14 DTS

Reply:
I removed it.
Thank you for your patience